lib/rust/metamodel/src/java/bincode.rs

```diff
@@ -197,6 +197,7 @@ impl DeserializerBuilder {
         method.static_ = true;
         method.body = body;
         method.arguments = vec![(message_ty, message.to_owned())];
+        method.visibility = None;
```

Comment:
Does `None` mean _package private_?

Reply:
This is in the docs for the `Method` type, but in general the types in `metamodel.java.syntax` correspond directly to Java syntax, so `None` indicates the absence of any visibility-modifier keyword.